### DISCUSSION TITLE
Use memories (documents) loaded by Kernel Memory

### DISCUSSION THREAD

**Discussion by maciejgos** [microsoft/semantic-kernel#3464] [Q&A]
I have a problem linking Semantic Kernel with Kernel Memory and use this as memories? In my implementation I load documents as batch process using Kernel Memory.

Another API tries to use memories prepared in the previous step as source to answer asked questions using this memory.

How could both projects be glued together to work? I will add more context my idea is to load data into vector DB using Kernel Memory and use this memory in TextMemoryPlugin.

**Comment by markwallace-microsoft** [comment]
I am cleaning up old unanswered discussions
